src/app/authentication/components/form-forgot-password/form-forgot-password.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { AuthenticationService } from 'app/authentication/services/authentication.service';
+
+@Component({
+  selector: 'app-form-forgot-password',
+  templateUrl: './form-forgot-password.component.html',
+  styleUrls: ['./form-forgot-password.component.scss']
+})
+export class FormForgotPasswordComponent implements OnInit {
+  forgotPasswordForm: FormGroup;
+  alertMessage = '';
+
+  constructor(private authenticationService: AuthenticationService) {
+    this.forgotPasswordForm = new FormGroup({
+      email: new FormControl('', [Validators.required, Validators.email])
+    });
+  }
+
+  ngOnInit(): void {
+  }
+
+  onSubmit(): void {
+    this.alertMessage = '';
```

Comment:
Do we need it?

Reply:
@junan IMO we need this, in terms of UX when the user already submits the form and the alert message still there when they resubmit the form, they might be confused and won't know that the second submission works or not. What do you think?
![user_forgot_password_form_submit](https://user-images.githubusercontent.com/1772999/104546298-6e289500-565e-11eb-93a8-c87e8802ee77.gif)